From: Ghislain Antony Vaillant <ghisvail@gmail.com>
Date: Wed, 15 Mar 2017 12:43:24 +0000
Subject: Use packaged docs

---
 docs/conf.py | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -27,8 +27,14 @@
 autodoc_member_order = 'bysource'
 
 intersphinx_mapping = {
-    'python': ('http://docs.python.org/3.5/', None),
-    'agate': ('http://agate.readthedocs.org/en/latest/', None)
+    'python': (
+        'http://docs.python.org/3/',
+        '/usr/share/doc/python3/html/objects.inv',
+    ),
+    'agate': (
+        'http://agate.readthedocs.org/en/latest/',
+        '/usr/share/doc/python-agate-doc/html/objects.inv',
+    ),
 }
 
 # Add any paths that contain templates here, relative to this directory.
